sdk/cognitiveservices/Azure.ResourceManager.CognitiveServices/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 1.1.0 (Unreleased)
+
+### Other Changes
+
+- Switch to API version `2022-10-01`
```

Comment:
please run `eng\scripts\Update-Snippets.ps1 cognitiveservices` to fix the ci failure

Reply:
I run `eng\scripts\Update-Snippets.ps1 cognitiveservices` locallly and I don't see any changes in repo

Reply:
run eng\scripts\Export-API.ps1 and got some changes